src/index.spec.js

```diff
@@ -37,6 +49,53 @@ describe('Dropzone', () => {
       expect(dropzone.html()).toMatchSnapshot();
     });
 
+    function collectEventListenerCalls(calls) {
```

Comment:
Please add a JSDoc comment for this function. I hardly understand what it does and why it's needed.